O/OpenSpiel/build_tarballs.jl

```diff
@@ -57,8 +56,11 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
+    BuildDependency(PackageSpec(name="libjulia_jll", version=julia_version)),
     Dependency("libcxxwrap_julia_jll")
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; preferred_gcc_version = v"8")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
+    preferred_gcc_version=v"9",
```

Comment:
```
% nm libspieljl.so|c++filt|grep "GLIBCXX_3.4.26"
                 U std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::operator std::basic_string_view<char, std::char_traits<char> >() const@@GLIBCXX_3.4.26
                 U std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::__sv_wrapper::__sv_wrapper(std::basic_string_view<char, std::char_traits<char> >)@@GLIBCXX_3.4.26
                 U std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::_S_to_string_view(std::basic_string_view<char, std::char_traits<char> >)@@GLIBCXX_3.4.26
                 U std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::data()@@GLIBCXX_3.4.26
                 U std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::basic_string(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >::__sv_wrapper, std::allocator<char> const&)@@GLIBCXX_3.4.26
                 U std::__cxx11::basic_stringstream<char, std::char_traits<char>, std::allocator<char> >::basic_stringstream()@@GLIBCXX_3.4.26
                 U std::__cxx11::basic_ostringstream<char, std::char_traits<char>, std::allocator<char> >::basic_ostringstream()@@GLIBCXX_3.4.26
```
I'm fairly sure this won't run at all in Julia v1.5.  Why don't you use _older_ versions of GCC, like 5 at least?

Reply:
The main reason is that it doesn't compile with an older version.

https://dev.azure.com/JuliaPackaging/Yggdrasil/_build/results?buildId=9294&view=logs&j=9f338677-a1f4-503e-790c-a58f3dc3b685&t=2483d461-f979-587b-ef3a-641fb5f6eb6a&l=5029

Reply:
> I'm fairly sure this won't run at all in Julia v1.5

I think this is fine since we're targeting Julia v1.6 for this package?

Reply:
Are we?  You're building against libjulia 1.5

Reply:
I see. Yeah, then I have no choice but to wait for 1.6 
╮(╯▽╰)╭

Reply:
With Julia v1.6 we ship a newer libstdc++ in the official binaries, so those symbols marked with `GLIBCXX_3.4.26` won't be a problem